pkg/neorpc/filters.go

```diff
@@ -20,6 +20,7 @@ type (
 	TxFilter struct {
 		Sender *util.Uint160 `json:"sender,omitempty"`
 		Signer *util.Uint160 `json:"signer,omitempty"`
+		Type   *string       `json:"type,omitempty"`
```

Comment:
We need to adjust documentation at `pkg/docs/notifications.md` (both for Notary request and transaction subscriptions).